javascript/ql/src/Security/CWE-020/UselessRegExpCharacterEscape.qhelp

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+
+	<overview>
+		<p>
+
+			Characters in string/template literals, and
+			regular expression literals can be backslash escaped by prepending a
```

Comment:
Should we be adding a hyphen here, i.e. `backlash-escaped`? (several instances of this in this file)

Reply:
Good idea. Done.